### PR TITLE
Change EC2 keyname reference in all environments

### PR DESCRIPTION
**Pull request by ahayden**
This will update the EC2 key used between Travis, the bastion, and db instances. The new private key is in the password vault under Shared-Admin/AWS Agora.

### CODE REVIEW

File: config/develop/agoradb.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 template_path: mongodb.yaml
 stack_name: agoradb-develop
 parameters:
-  KeyPairName: 'toptal'
+  KeyPairName: 'agora-travis'
```
Comment (zaro0508) [inline config/develop/agoradb.yaml:4]: the db is not exposed to travis.  maybe we should rename the key to not be travis specific?

File: config/staging/agora.yaml [hunk 2]
```diff
@@ -13,6 +13,6 @@ parameters:
   SnsNotificationEndpoint: 'agora-staging@sagebase.org'
   EbSolutionStackName: '64bit Amazon Linux 2018.03 v4.13.0 running Node.js'
   EC2InstanceType: 't3.medium'
-  EC2KeyName: 'toptal'
+  EC2KeyName: 'agora-travis'
```
Comment (zaro0508) [inline config/staging/agora.yaml:16]: these app servers are not exposed to travis either.

File: config/develop/agoradb.yaml (hunk 1 above)
Comment (ahayden) [reply config/develop/agoradb.yaml:4]: Does the db instance need to have that key associated with it at all? I should just remove it if not.

File: config/staging/agora.yaml (hunk 2 above)
Comment (ahayden) [reply config/staging/agora.yaml:16]: Also with the app servers. If they don't need this key, I'll just remove it from the instance.

File: config/develop/agoradb.yaml (hunk 1 above)
Comment (zaro0508) [reply config/develop/agoradb.yaml:4]: only if we need to debug it which we did in the early days of development but maybe not now.  meh, you can decide on this one.

File: config/staging/agora.yaml (hunk 2 above)
Comment (zaro0508) [reply config/staging/agora.yaml:16]: yes, we do need to access app servers for debugging.

**Comment by ahayden** [comment]
I'm closing this PR because the changes are included in https://github.com/Sage-Bionetworks/Agora-infra/pull/82

**Pull request closed without merging**